filetests/isa/x86/insertlane-run.clif

```diff
@@ -0,0 +1,46 @@
+test run
+set enable_simd
+
+function %test_insertlane_b8() -> b8 {
+ebb0:
+    v0 = bconst.b8 false
+    v1 = bconst.b8 true
+    v2 = splat.b8x16 v0
+    v3 = insertlane v2, 10, v1
+    v4 = extractlane v3, 10
```

Comment:
This test is dependent on splat and extractlane working. Can we try to reduce the dependency by only depending on vconst, and use vconst as the input and expected values, instead?

Reply:
Well, I can't compare vconsts yet but I am working on it in a separate branch: why don't I change the initial `splat` to a `vconst` and add a TODO to replace the extractlane + comparison with a direct vector comparison?

Reply:
Yes, that's great, thanks.